.github/workflows/ci.yml

```diff
@@ -63,27 +78,38 @@ jobs:
             bin/Release/*.TapPackage
 
   UnitTests:
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, Windows]
```

Comment:
Can we switch this back to Linux runners?

Reply:
restored back